tutorials/preprocessing/15_handling_bad_channels.py

```diff
@@ -239,7 +239,7 @@
 
 # %%
 # Note that we used the ``exclude=[]`` trick in the call to
-# :meth:`~mne.io.Raw.pick_types` to make sure the bad channels were not
+# :meth:`pick()` method from the instance of `~mne.io.Raw` to make sure the bad channels were not
```

Comment:
```suggestion
# :meth:`~mne.io.Raw.pick()` to make sure the bad channels were not
```

The `~` sign makes it so the `mne.io.Raw.` part doesn't show in the docs, but is present in the hyperlink.

Reply:
Cool